pkgs/http/lib/src/io_client.dart

```diff
@@ -46,6 +47,22 @@ class _ClientSocketException extends ClientException
   String toString() => 'ClientException with $cause, uri=$uri';
 }
 
+class _IOStreamedResponseV2 extends IOStreamedResponse
+    with BaseResponseWithUrl {
+  @override
+  final Uri url;
+
+  _IOStreamedResponseV2(super.stream, super.statusCode,
+      {super.contentLength,
+      super.request,
+      super.headers,
+      super.isRedirect,
+      required this.url,
+      super.persistentConnection,
+      super.reasonPhrase,
+      super.inner});
```

Comment:
Consider ordering these parameters in some consistent way.
If nothing else, because they are ordered in a way I can't see, consider putting the required parameter first.
(Maybe it's just because GitHub shows `required` as red that it stands out.)

Reply:
I put the required parameter first.